js/views/folder.js

```diff
@@ -83,10 +82,15 @@ views.Folders = Backbone.Marionette.CollectionView.extend({
 		var activeFolder = activeAccount;
```

Comment:
You'll probably want to use the mail delimiter on the line above this one too (`folderId.split()`)